src/OrganisationUnitTree/shouldIncludePath.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Checks if the "path" is either inside includedPaths
+ * or is a parent path of one of the includedPaths
```

Comment:
Perhaps this is being checked implicitely, but when I see the code, it seems to be only doing what's on line 2, I don't see it checking a parent path. If this code is functioning correctly, I'd like to see a different explation in the comment.

Reply:
I think this is easier to visualize with an example:

```js
const includePaths = [ 'foo/bar/baz' ]
const path = 'foo/bar/baz'
// In the first iteration of "some", this would be the first check
includedPaths[0].indexOf(path) === 0 // true

// vs

// Here the "path" is a parent path of the first item in includePaths
// and because it's a substring at position 0, the condition returns true as well
const includePaths = [ 'foo/bar/baz' ]
const path = 'foo/bar'
// In the first iteration of "some", this would be the first check
includedPathd[0].indexOf(path) === 0 // true
```

Reply:
Maybe a test would be much better than a comment.
What do you think?